source/1_engine/rex_engine/src/diagnostics/logging/logger_config.cpp

```diff
@@ -114,11 +121,24 @@ namespace rexlog
           }
         }
 
-        details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
+        rexlog::details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
       }
 
     } // namespace helpers
-  }   // namespace cfg
-} // namespace rexlog
 
-// NOLINTEND(misc-definitions-in-headers)
+    // search for REXLOG_LEVEL= in the args and use it to init the levels
+    void load_log_levels(const rsl::vector<rsl::string_view>& args)
+    {
+      const rex::DebugString rexlog_level_prefix("REXLOG_LEVEL=");
+
+      for(rsl::string_view s : args)
+      {
+        if(s.find(rexlog_level_prefix) == 0)
```

Comment:
note: we do have rsl::string_type::contains but in this case not sure if you should use it, as it could introduce a bug if the cli doesn't start with "REXLOG_LEVEL="